packages/image/README.md

```diff
@@ -0,0 +1,82 @@
+# @rollup/plugin-image
```

Comment:
Should the README include badges at the top like [the other packages](https://raw.githubusercontent.com/rollup/plugins/master/packages/buble/README.md)?

Reply:
Yes, yes it should.